content/en/real_user_monitoring/mobile_and_tv_monitoring/setup/otel.md

```diff
@@ -1,7 +1,14 @@
 ---
-title: OpenTelemetry Support
+title: RUM OpenTelemetry Setup
```

Comment:
It seems this OTel setup is specific to flutter, and shouldn't appear as a top level application type. It would make pre sense for it to appear as a sub page of Flutter, or be included in Flutter>Advanced config page.

Reply:
ah ok, will move, thanks @xgouchet !